### PR TITLE
More exciting error messages

### PR DESCRIPTION
**Pull request by MandL27**
Going along with the whole PMD theme here.

### CODE REVIEW

File: Server/Network/MessageProcessor.cs [hunk 1]
```diff
@@ -231,7 +231,7 @@ internal static void ProcessData(Client client, string[] parse)
                                 }
                                 else
                                 {
-                                    Messenger.PlainMsg(client, "Character does not exist", Enums.PlainMsgType.Chars);
+                                    Messenger.PlainMsg(client, "This character hasn't been sent in yet!", Enums.PlainMsgType.Chars);
```
Comment (pzaps) [inline Server/Network/MessageProcessor.cs]: Can this line be more descriptive? I think new players will be confused by what this actually means

File: Server/Network/MessageProcessor.cs [hunk 2]
```diff
@@ -390,11 +390,11 @@ internal static void ProcessData(Client client, string[] parse)
                                 {
                                     PlayerManager.DeleteCharacter(dbConnection, client.Player.AccountName, charNum);
                                     Messenger.SendChars(dbConnection, client);
-                                    Messenger.PlainMsg(client, "Character has been deleted!", Enums.PlainMsgType.Chars);
+                                    Messenger.PlainMsg(client, "The character has been sent back from the Pokémon world, never to return.", Enums.PlainMsgType.Chars);
```
Comment (pzaps) [inline Server/Network/MessageProcessor.cs:393]: Same with this one, you can recreate characters!

File: Server/Network/MessageProcessor.cs (hunk 1 above)
Comment (MandL27) [inline Server/Network/MessageProcessor.cs]: Good point. We should probably revert that.

File: Server/Network/MessageProcessor.cs (hunk 2 above)
Comment (MandL27) [inline Server/Network/MessageProcessor.cs:393]: Yeah, but it'll have lost all its progress. It's like having CharacterName I and CharacterName II.

**Pull request merged**